third-party/realdds/src/dds-stream.cpp

```diff
@@ -128,6 +128,29 @@ std::shared_ptr< dds_topic > const & dds_stream::get_topic() const
     DDS_THROW( runtime_error, "stream '" + name() + "' must be open to get_topic()" );
 }
 
+std::vector<std::string> dds_stream::get_post_processing_filters_names() const
+{
+    std::vector<std::string> filter_names;
+    if (_name == "Depth")
+    {
+        filter_names.push_back("Decimation Filter");
```

Comment:
Post processing is a LibRS concept, not realdds. This list should be in SDK layer.
I think no changes needed at realdds level. SDK can not use this if not needed https://github.com/IntelRealSense/librealsense/blob/development/src/dds/rs-dds-device-proxy.cpp#L301

Reply:
Great comment! Definitely needs to be in LRS rather than realdds - will correct it.

Reply:
done